plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -10,7 +10,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/httpcore-4.3.3.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jboss-dmr-1.2.0.Final.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/log4j-1.2.17.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-1.0.0-SNAPSHOT.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-1.0.0-SNAPSHOT.jar" sourcepath="/openshift-restclient-java/src/main/java"/>
```

Comment:
I guess that we should bump the required client version since this code change in JBT requires a client that supports ImageStreams?